mesonbuild/interpreter/interpreter.py

```diff
@@ -1331,7 +1331,7 @@ def add_languages_for(self, args: T.List[str], required: bool, for_machine: Mach
         # compilers we don't add anything for cython here, and instead do it
         # When the first cython target using a particular language is used.
         if 'vala' in langs and 'c' not in langs:
-            FeatureNew.single_use('Adding Vala language without C', '0.59.0', self.subproject)
+            FeatureNew.single_use('Adding Vala language without C', '0.59.0', self.subproject, location=node)
```

Comment:
I don't see where the `node` is coming from here, or in the next use?

Reply:
This one was broken, that particular function doesn't have a node kwarg -- the next use does though...